crates/services/gas_price_service/src/v1/uninitialized_task.rs

```diff
@@ -189,7 +189,11 @@ where
             self.block_stream,
         );
 
-        let recorded_height = self.gas_price_db.get_recorded_height()?;
+        let recorded_height = match self.gas_price_db.get_recorded_height()? {
+            Some(height) => height,
+            None => BlockHeight::from(latest_block_height),
+        };
```

Comment:
nit (see https://www.lurklurk.org/effective-rust/transform.html)

```suggestion
        let recorded_height = self
            .gas_price_db
            .get_recorded_height()?
            .unwrap_or(BlockHeight::from(latest_block_height));
```

Reply:
addressed in 4ca7a9455f